lustrefs-exporter/src/main.rs

```diff
@@ -179,11 +199,28 @@ async fn scrape(Query(params): Query<Params>) -> Result<Response<Body>, Error> {
 
     output.append(&mut lnetctl_stats_record);
 
-    let lustre_stats = build_lustre_stats(output);
+    let opentelemetry_metrics = OpenTelemetryMetrics::new(meter.clone());
+
+    let mut lustre_stats = vec![];
+    let encoder = TextEncoder::new();
+    let metric_families = registry.gather();
+    let _encoder_results = encoder.encode(&metric_families, &mut lustre_stats);
+
+    // Build OTEL metrics
+    openmetrics::build_lustre_stats(&output, opentelemetry_metrics);
+
+    let mut buffer = vec![];
+    let encoder = TextEncoder::new();
+    let metric_families = registry.gather();
+    encoder.encode(&metric_families, &mut buffer).unwrap();
+
+    let lustre_stats = String::from_utf8_lossy(&buffer).to_string();
 
     let body = if let Some(stream) = jobstats {
-        let merged =
-            tokio_stream::StreamExt::chain(tokio_stream::once(Ok(lustre_stats.into())), stream);
+        let merged = tokio_stream::StreamExt::chain(
+            tokio_stream::once(Ok::<_, Infallible>(lustre_stats)),
+            tokio_stream::once(Ok::<_, Infallible>(stream)),
+        );
```

Comment:
```suggestion
        let merged = tokio_stream::once(Ok::<_, Infallible>(lustre_stats))
            .chain(tokio_stream::once(Ok(stream)));
```

Reply:
If we want to add more, we can then tack a `.chain` after. The alternative is chain(chain(A, B), C)